src/SDK/Trace/Behavior/HttpSpanExporterTrait.php

```diff
@@ -46,14 +45,20 @@ protected function doExport(iterable $spans): int
         try {
             $response = $this->dispatchSpans($spans);
         } catch (ClientExceptionInterface $e) {
+            $this->error('Error exporting span', ['error' => $e->getMessage()]);
```

Comment:
Why don't you add the stack trace here? It has been generated anyway.

Reply:
done. I wasn't sure what the performance penalty might be for sending the entire exception.

Reply:
The biggest performance penalty is generating the stack trace in the first place, however some "log inspection tools" will provide additional features, if the stack trace is provided.